pkg/resources/api_integration.go

```diff
@@ -127,143 +130,186 @@ func APIIntegration() *schema.Resource {
 	}
 }
 
+func toApiIntegrationEndpointPrefix(paths []string) []sdk.ApiIntegrationEndpointPrefix {
+	allowedPrefixes := make([]sdk.ApiIntegrationEndpointPrefix, len(paths))
+	for i, prefix := range paths {
+		allowedPrefixes[i] = sdk.ApiIntegrationEndpointPrefix{Path: prefix}
+	}
+	return allowedPrefixes
+}
+
 // CreateAPIIntegration implements schema.CreateFunc.
 func CreateAPIIntegration(d *schema.ResourceData, meta interface{}) error {
```

Comment:
jan was using createcontext function in his stage pr, I thought that was a pretty good idea. i don't think its necessary to do in these prs now, but its something we should eventually do for all resources as part of a larger refactoring effort

Reply:
Added as a part of SNOW-867235.